apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/LimitPriceWarning/LimitPriceWarning.stories.tsx

```diff
@@ -23,7 +25,14 @@ const Initial = () => {
             <Button
               title="Open Modal"
               onPress={() => {
-                content !== undefined ? closeModal() : openModal(strings.limitPriceWarningTitle, <LimitPriceWarning />)
+                content !== undefined
+                  ? closeModal()
+                  : openModal(
+                      strings.limitPriceWarningTitle,
+                      <SelectedWalletProvider wallet={walletMocks.wallet}>
```

Comment:
@stackchain another potential solution! i will try to apply this to the cancellation tx fee

Reply:
ctx won't be the same, the issue is the funnel on modal.